src/dvclive/studio.py

```diff
@@ -24,32 +25,47 @@ def _cast_to_numbers(datapoints):
     return datapoints
 
 
-def _to_dvc_format(plots):
-    formatted = {}
-    for k, v in plots.items():
-        formatted[k] = {"data": v}
-    return formatted
+def _rel_path(path, dvc_root_path):
+    absolute_path = Path(path).resolve()
+    return str(absolute_path.relative_to(dvc_root_path).as_posix())
```

Comment:
@mvshmakov 

I think the `as_posix` here actually fixes an existing bug never discovered, as updates from a windows machine were being sent with paths in windows format.
I believe Studio was failing to match those, but didn't actually tested from windows.